Portal/src/Datahub.Portal/Pages/Public/RegisterPage.razor

```diff
@@ -111,16 +131,22 @@
     private async Task DisplayFailedMessage(int time = 15)
     {
         _failed = false;
-        StateHasChanged();
+        await InvokeAsync(StateHasChanged);
         await Task.Delay(time * 1000);
         _failed = true;
-        StateHasChanged();
+        await InvokeAsync(StateHasChanged);
```

Comment:
@davidreneuw I think this also fixes the registration bug

Reply:
I think so